python/tests/conftest.py

```diff
@@ -1149,29 +1090,14 @@ def object_storage_uri_incl_bucket(request):
 @pytest.fixture(
     scope="function",
     params=(
-        [
-            "lmdb_version_store_v1",
-            "lmdb_version_store_v2",
-            "s3_version_store_v1",
-            "s3_version_store_v2",
-            "in_memory_version_store",
-            pytest.param(
-                "azure_version_store",
-                marks=pytest.mark.skipif(MACOS_CONDA_BUILD, reason=MACOS_CONDA_BUILD_SKIP_REASON),
-            ),
-            pytest.param(
-                "mongo_version_store",
-                marks=pytest.mark.skipif(sys.platform != "linux", reason="The mongo store is only supported on Linux"),
-            ),
-            pytest.param(
-                "real_s3_version_store",
-                marks=pytest.mark.skipif(
-                    not PERSISTENT_STORAGE_TESTS_ENABLED, reason="Can be used only when persistent storage is enabled"
-                ),
-            ),
-        ]
-        if not MACOS_CONDA_BUILD
-        else ["lmdb_version_store_v1", "lmdb_version_store_v2", "s3_version_store_v1", "s3_version_store_v2"]
+        "lmdb_version_store_v1",
+        "lmdb_version_store_v2",
+        "s3_version_store_v1",
+        "s3_version_store_v2",
```

Comment:
Unrelated to this PR, but I think we should drop the `s3_version_store_v2` fixture from these lists. Doing one encoding on S3, and both on LMDB (which is much faster) ought to be fine

Reply:
I can mark it with a `thorough` custom `mark`, which will make it easy to skip from the command line at a later date.